spring-data-jpa-batch-dml/src/main/java/com/baeldung/jpa/entity/Customer.java

```diff
@@ -0,0 +1,60 @@
+package com.baeldung.jpa.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+/**
+ * Customer Entity class
+ * @author ysharma2512
+ *
+ */
+@Entity
+public class Customer {
+
+    @Id
```

Comment:
The file here feels a little disorganized. It starts with a field, and then all the setters, followed by fields, and then the constructor. Could we rearrange this to have a more typical setup?